File: DataIngestion/models/datapoint.py
from dataclasses import dataclass
from datetime import date

@dataclass
class DataPoint:
	open: float
	high: float
	low: float
	close: float
	adjusted_close: float
	volume: int
	dividend_amount: float
	split_coefficient: float
	date_: date

TimeSeries = list[DataPoint]

File: StockScreenerWebApp/Models/StockScreenerDb/DataPoint.cs
namespace StockScreenerWebApp.Models.StockScreenerDb
{
    public partial class DataPoint
    {
        public string Symbol { get; set; } = null!;
        public DateTime Date { get; set; }
        public double Open { get; set; }
        public double High { get; set; }
        public double Low { get; set; }
        public double Close { get; set; }
        public double AdjustedClose { get; set; }
        public long Volume { get; set; }
        public double DividendAmount { get; set; }
        public double SplitCoefficient { get; set; }

        public virtual Symbol SymbolNavigation { get; set; } = null!;
    }
}
